R/recipe.R

```diff
@@ -792,13 +798,13 @@ required_pkgs.recipe <- function(x, infra = TRUE, ...) {
   res
 }
 
-#' @rdname required_pkgs
+#' @rdname required_pkgs.recipe
```

Comment:
I think in your PR this still is `@rdname required_pkgs`, but I think that it has to link to the documentation for the specific recipe method to work right